### ISSUE TITLE
notation in latex can have runon lines

### ISSUE THREAD

**Issue by kcrisman** [rbeezer/mathbook#446]
If you let the description of a piece of notation get really  long, in the html it looks ok (not great, but you get multiple lines).  In LaTeX it just runs off the edge of the page.  (In fact, it makes the entire list of page numbers zoom off the right side too.)  Of course, maybe the descriptions shouldn't be long, but this should still fail gracefully.

Example:

```
<notation><usage>\gamma</usage><description>Euler-Mascheroni gamma constant limit of difference between the harmonic series and natural logarithm</description></notation>
```

**Comment by rbeezer** [comment]
Thanks, KDC!  That is all a LaTeX table, so the middle part, the description should probably be put into `\parbox`, or the column should have fixed-width, or similar.

**Comment by rbeezer** [comment]
Fixed up for both LaTeX and HTML at 9970852a

Needs some CSS, recorded at #470
